src/manifestManager.ts

```diff
@@ -150,9 +150,7 @@ export class ManifestManager implements vscode.Disposable {
         }
 
         // Ensure that build directory of active manifest exists
-        if (!(await exists(manifest.buildDir))) {
-            await fs.mkdir(manifest.buildDir)
-        }
+        await fs.mkdir(manifest.buildDir, { recursive: true })
```

Comment:
Wouldn't this fail if the dir exists already?

Reply:
According to the [docs](https://nodejs.org/api/fs.html#fspromisesmkdirpath-options), it won't. I also tested it